spec/bolt/node/winrm_spec.rb

```diff
@@ -186,18 +184,31 @@ def stub_winrm_to_raise(klass, message)
         ).value
       ).to eq(<<SHELLWORDS)
 echo $env:path\r
-\r
 SHELLWORDS
     end
   end
 
+  it "does not deadlock scripts that write > 4k to stderr", winrm: true do
```

Comment:
This test is derived from one in the powershell module... and there are other similar ones that could come along for the ride here if we'd like to add them.

https://github.com/puppetlabs/puppetlabs-powershell/blob/master/spec/integration/puppet_x/puppetlabs/powershell_manager_spec.rb#L520-L532